neo4j-to-neptune/docs/conversion-config.md

```diff
@@ -66,120 +62,187 @@ skipVertices:
 ## Label Mapping
 
 ### Vertex Labels
-
 - Neo4j vertices can have multiple labels separated by colons (e.g., `Person:Employee`)
 - Each individual label is mapped independently if a mapping exists
 - Unmapped labels are kept as-is
 - The output uses semicolons as separators (Neptune format)
 
-**Example:**
-- Input: `Person:Employee` with mapping `Person: Individual`
-- Output: `Individual;Employee`
+Map Neo4j node labels to Neptune-compatible vertex labels:
 
-### Edge Labels
+```yaml
+vertexLabels:
+  Person: Individual
+  Company: Organization
+  User: Customer
+  Account: Profile
+```
 
+### Edge Labels
 - Neo4j relationships have a single type/label
 - The entire label is mapped if a mapping exists
 - Unmapped labels are kept as-is
 
+Map Neo4j relationship types to Neptune-compatible edge labels:
+
+```yaml
+edgeLabels:
+  WORKS_FOR: EMPLOYED_BY
+  KNOWS: CONNECTED_TO
+  MANAGES: SUPERVISES
+  OWNS: POSSESSES
+```
 **Example:**
 - Input: `WORKS_FOR` with mapping `WORKS_FOR: EMPLOYED_BY`
 - Output: `EMPLOYED_BY`
 
-## Record Filtering
-
-### Skip Vertices
-
-You can skip vertices in two ways:
-
-1. **By ID**: Skip specific vertices by their unique identifiers (note that IDs are treated as strings)
-2. **By Label**: Skip all vertices that have any of the specified labels
-
-When a vertex is skipped:
-- The vertex is not written to the output vertices CSV file
-- All edges connected to that vertex (incoming and outgoing) are automatically skipped
-- The skipped vertex ID is tracked for edge filtering
+**Notes:**
+- Label mappings are applied before ID transformations
+- Unmapped labels remain unchanged
+- Case-sensitive matching
 
-### Skip Edges
+## ID Transformation
 
-You can skip edges in following way:
+Transform vertex and edge IDs using template patterns that reference original data fields.
 
-1. **By Label**: Skip all edges of the specified relationship types
+### Vertex ID Transformation
 
-_**Note:** label only for edge as Neo4j default csv export doesn't include original edge IDs._
+Transform vertex IDs using templates that can reference:
 
-### Automatic Edge Filtering
+| Placeholder | Description | Example Value |
+|-------------|-------------|---------------|
+| `{_id}` | Original Neo4j vertex ID | `"123"` |
+| `{_labels}` | Vertex labels (colon-separated → underscore-separated) | `"Person_Employee"` |
```

Comment:
what's `colon-separated` referring to here? Looks like it's underscore-separated in both cases